src/main/groovy/life/qbic/datamodel/dtos/business/CostEstimate.groovy

```diff
@@ -1,37 +1,33 @@
 package life.qbic.datamodel.dtos.business
 
-import groovy.transform.EqualsAndHashCode
-import life.qbic.datamodel.dtos.general.Person
 
 /**
- * A cost estimate Dto for a project
+ * A cost estimate for a project
  *
  * During project planning a cost estimate is often required to list expected costs.
  * This estimate is not intended to be legally binding. It provides an overview over the project
  * scope and the costs associated with it.
  *
  * @since: 1.12.0
- *
  */
-@EqualsAndHashCode
```

Comment:
why deleting this?

Reply:
Oversight during merge conflict. Nice Catch!